sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/CommandUtils.java

```diff
@@ -1,20 +1,23 @@
 package com.mesosphere.sdk.offer;
 
-import org.apache.mesos.Protos;
 import com.mesosphere.sdk.specification.CommandSpec;
+import org.apache.mesos.Protos;
+
+import java.util.Map;
 
 /**
  * This class provides utility methods for the construction of {@link org.apache.mesos.Protos.CommandInfo} protobufs
  * from {@link CommandSpec}s.
  */
 public class CommandUtils {
 
-    public static Protos.CommandInfo addEnvVar(
-            Protos.CommandInfo command,
+    public static Protos.CommandInfo setEnvVar(
```

Comment:
Why return a CommandInfo rather than a CommandInfo.Builder when you take the latter?